<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ukliw</class>
 <widget class="QWidget" name="Ukliw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>14</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="descr">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>34</y>
     <width>205</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="size">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>14</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>593</x>
     <y>23</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="installedVersion">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>14</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="candidateVersion">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>34</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDetail">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>23</y>
     <width>40</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="installedsize">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>34</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rating">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>34</y>
     <width>23</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ratingtext">
   <property name="geometry">
    <rect>
     <x>528</x>
     <y>34</y>
     <width>20</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
